### ISSUE TITLE
Issues with mamba (possibly temporary?)

### ISSUE THREAD

**Issue by tlambert03** [conda-incubator/setup-miniconda#217]
posting as a courtesy, not sure if this is a bug here or in mamba

I'm seeing the following error in [one of my ci runs](https://github.com/tlambert03/psygnal/runs/5157480532?check_suite_focus=true#step:3:565).  This appears to be the same as https://github.com/mamba-org/mamba/issues/1480 ... which was closed by https://github.com/mamba-org/mamba/pull/1460  But the mamba issue was closed 2 days ago and this error just happened:

```
Run conda-incubator/setup-miniconda@v2
Gathering Inputs...
Creating bootstrap condarc file in /home/runner/.condarc...
Ensuring installer...
Running installer...
Setup environment variables...
Parsing environment...
Configuring conda package cache...
Applying initial configuration...
Initializing conda shell integration...
Adding tools to 'base' env...
Ensuring environment...
Updating 'test' env from conda create (simple)...
  /usr/share/miniconda3/condabin/mamba create --name test
  Warning: 
  # >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<
  
      Traceback (most recent call last):
        File "/usr/share/miniconda3/lib/python3.9/site-packages/conda/exceptions.py", line [10](https://github.com/tlambert03/psygnal/runs/5157480532?check_suite_focus=true#step:3:10)80, in __call__
          return func(*args, **kwargs)
        File "/usr/share/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 934, in exception_converter
          raise e
        File "/usr/share/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 928, in exception_converter
          exit_code = _wrapped_main(*args, **kwargs)
        File "/usr/share/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 882, in _wrapped_main
          result = do_call(args, p)
        File "/usr/share/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 748, in do_call
          exit_code = create(args, parser)
        File "/usr/share/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 624, in create
          return install(args, parser, "create")
        File "/usr/share/miniconda3/lib/python3.9/site-packages/mamba/mamba.py", line 577, in install
          if not transaction.prompt():
      UnboundLocalError: local variable 'transaction' referenced before assignment
  
  `$ /usr/share/miniconda3/condabin/mamba create --name test`
  
    environment variables:
                   CIO_TEST=<not set>
                      CONDA=/usr/share/miniconda3
             CONDA_PKGS_DIR=/home/runner/conda_pkgs_dir
                 CONDA_ROOT=/usr/share/miniconda3
             CURL_CA_BUNDLE=<not set>
        DEPLOYMENT_BASEPATH=/opt/runner
          GITHUB_EVENT_PATH=/home/runner/work/_temp/_github_workflow/event.json
                GITHUB_PATH=/home/runner/work/_temp/_runner_file_commands/add_path_2bbc4f3d-88a5-4
                            e15-94ed-afb05d99e456
                       PATH=/usr/share/miniconda3/condabin:/home/linuxbrew/.linuxbrew/bin:/home/li
                            nuxbrew/.linuxbrew/sbin:/home/runner/.local/bin:/opt/pipx_bin:/home/ru
                            nner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.g
                            hcup/bin:/home/runner/.dotnet/tools:/snap/bin:/usr/local/sbin:/usr/loc
                            al/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap
                            /bin
         REQUESTS_CA_BUNDLE=<not set>
          SELENIUM_JAR_PATH=/usr/share/java/selenium-server.jar
              SSL_CERT_FILE=<not set>
                 SWIFT_PATH=/usr/share/swift/usr/bin
  
       active environment : None
         user config file : /home/runner/.condarc
   populated config files : /usr/share/miniconda3/.condarc
                            /home/runner/.condarc
            conda version : 4.[11](https://github.com/tlambert03/psygnal/runs/5157480532?check_suite_focus=true#step:3:11).0
      conda-build version : not installed
           python version : 3.9.10.final.0
         virtual packages : __linux=5.11.0=0
                            __glibc=2.31=0
                            __unix=0=0
                            __archspec=1=x86_64
         base environment : /usr/share/miniconda3  (writable)
        conda av data dir : /usr/share/miniconda3/etc/conda
    conda av metadata url : None
             channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                            https://conda.anaconda.org/conda-forge/noarch
            package cache : /home/runner/conda_pkgs_dir
         envs directories : /usr/share/miniconda3/envs
                            /home/runner/.conda/envs
                 platform : linux-64
               user-agent : conda/4.11.0 requests/2.27.1 CPython/3.9.10 Linux/5.11.0-1028-azure ubuntu/20.04.3 glibc/2.31
                  UID:GID : 1001:[12](https://github.com/tlambert03/psygnal/runs/5157480532?check_suite_focus=true#step:3:12)1
               netrc file : None
             offline mode : False
```

**Comment by yardasol** [comment]
I'm also having this issue. Looks like the fix for the `UnboundLocalError` was [published a couple hours ago in 0.21.1](https://github.com/mamba-org/mamba/releases/tag/2022.02.11). I think once those changes trickle up to `conda-forge`, this will fix itself (knock on wood), but right now it looks like 0.21.1 still isn't on conda-forge:
```
 $ mamba repoquery search "mamba"

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (0.17.0) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


Executing the query mamba

pkgs/main/linux-64       Using cache
pkgs/main/noarch         Using cache
pkgs/r/linux-64          Using cache
pkgs/r/noarch            Using cache
conda-forge/linux-64     Using cache
conda-forge/noarch       Using cache


 Name  Version Build           Channel             
────────────────────────────────────────────────────
 mamba 0.21.0  py39hfa8f2c8_0  conda-forge/linux-64
 mamba 0.21.0  py38h1abaa86_0  conda-forge/linux-64
 mamba 0.21.0  py37h6dacc13_0  conda-forge/linux-64
 mamba 0.21.0  py37h47bf687_0  conda-forge/linux-64
 mamba 0.21.0  py310hf87f941_0 conda-forge/linux-64
 mamba 0.20.0  py310hf87f941_0 conda-forge/linux-64
 mamba 0.20.0  py39hfa8f2c8_0  conda-forge/linux-64
.
.
.
```

**Comment by tlambert03** [comment]
k, i'll close this then!  thanks
